### PR TITLE
Misc. menu formatting improvements

### PR DESCRIPTION
**Pull request by ceski-1**
Any other requests for small tweaks that make sense to add to this PR?

### CODE REVIEW

**Comment by rfomin** [comment]
> Any other requests for small tweaks that make sense to add to this PR?

Maybe we should rename translucency options?

image

**Comment by rfomin** [comment]
How about replacing all the "yes/no" switches with "on/off"?

**Comment by fabiangreffrath** [comment]
> Maybe we should rename translucency options?

What do you suggest?

> How about replacing all the "yes/no" switches with "on/off"?

Why did we have both variants anyway?

File: src/m_menu.c [hunk 1]
```diff
@@ -4150,40 +4150,40 @@ setup_menu_t gen_settings4[] = {
...
-  {"Smooth Pixel Scaling", S_YESNO, m_null, M_X, M_Y,
+  {"Smooth Pixel Scaling", S_ONOFF, m_null, M_X, M_Y,
    {"smooth_scaling"}, 0, M_ResetScreen},
 
-  {"Sprite Translucency", S_YESNO|S_STRICT, m_null, M_X, M_SPC, {"translucency"}},
+  {"Sprite Translucency", S_ONOFF|S_STRICT, m_null, M_X, M_SPC, {"translucency"}},
 
-  {"Translucency", S_THERMO|S_ACTION|S_PCT, m_null, M_X_THRM8, M_SPC,
+  {"Translucency Filter", S_NUM|S_ACTION|S_PCT, m_null, M_X, M_SPC,
    {"tran_filter_pct"}, 0, M_Trans},
```
Comment (ceski-1) [inline src/m_menu.c:4170]: Okay, controversial change:

woof0019


Reasoning:
- Boom docs refer to this as "Translucency Filter". "Translucency" alone is simple but too vague in this context.
- The name "Translucency Filter" is too long to use with a thermo.
- The thermo is too small to use the mouse to return the value to 66%, which is poor usability.
- A bigger thermo would look out of place.
- There's no need to draw attention to this setting compared to everything else on the "Display" tab.
- Based on a quick Doomworld search, the sprite translucency toggle is far more important than fine adjustments to the opacity for walls/sprites.

Comment (fabiangreffrath) [reply src/m_menu.c:4170]: Sounds legit.

File: src/m_menu.c [hunk 2]
```diff
@@ -159,7 +159,7 @@ backdrop_t menu_backdrop;
 #define M_X_THRM11    (M_X - (M_THRM_SIZE11 + 3) * M_THRM_STEP)
 #define M_THRM_TXT_OFFSET 3
 #define M_THRM_SPC    (M_THRM_HEIGHT + 1)
-#define M_THRM_UL_VAL 20
+#define M_THRM_UL_VAL 50
```
Comment (ceski-1) [inline src/m_menu.c:162]: https://github.com/fabiangreffrath/woof/issues/1515

File: src/m_menu.c (hunk 1 above)
Comment (rfomin) [reply src/m_menu.c:4170]: I like the change.

**Review by rfomin — APPROVED** [review]

**Pull request merged**